testdata/BUSL-1.1.t1

```diff
@@ -0,0 +1,95 @@
+50.8%
```

Comment:
We want to get this percentage up before merging. This was the base case where I could get the LRE matching on everything under the hashicorp parameters